HACKING.md

```diff
@@ -164,6 +166,18 @@ reason we create the app first is that otherwise OpenShift will
 automatically instantiate Jenkins with default parameters when creating
 the Jenkins pipeline).
 
+Next, let's update the input ImageStreamTag for our jenkins deployment
+to match latest openshift (`jenkins:2`). Some older versions of the
+template in openshift uses `jenkins:latest`. This will no longer be
+needed once we are running on a newer version of OpenShift than 3.6 in
+CentOS CI. See [#32](https://github.com/coreos/fedora-coreos-pipeline/pull/32)
+and [#70](https://github.com/coreos/fedora-coreos-pipeline/pull/70)
+for context on why this may be needed):
```

Comment:
This is super minor/OCD, though can we consistently capitalize Jenkins and OpenShift? :)

Reply:
done - let me know if you spot any I missed